### DISCUSSION TITLE
Tests - not generating

### DISCUSSION THREAD

**Discussion by chyde80** [apideck-libraries/portman#451] [Q&A] [answered]
Hi all!  I'm doing a PoC with portman and love the promise of this tool. However, I cannot get tests to generate. I don't see any problems in the command line after executing and it does seem 'includeTests' is true in the CLI summary. I don't know where to start! The collections do generate but tests are empty. To make sure I wasn't just missing viewing them, I set 'bundleContractTests' to true and indeed it's empty.

**Comment by thim81** [comment, marked as answer]
Hi @chyde80 

Have you defined which type of tests you want to have generated?

Have a look at the example: https://github.com/apideck-libraries/portman/blob/main/examples/testsuite-contract-tests/portman-config.crm.json

**Reply by chyde80** [reply]
Thanks, would this work to just generate for everything defined in the openapi spec?
`"openApiOperation": "*::/*"`

**Reply by chyde80** [reply]
Got it fixed. I was passing in a config but neglected to include the default config settings - copied those into the config files and works like a charm. Thank you! Simple oversight.
